manifests/cli/params.pp

```diff
@@ -15,35 +15,36 @@
     'RedHat': {
       # We use system packages except on EL7
       if versioncmp($facts['os']['release']['major'], '8') >= 0 {
-        $hammer_plugin_prefix = 'rubygem-hammer_cli_'
+        $_hammer_plugin_prefix = 'rubygem-hammer_cli_'
       } else {
-        $hammer_plugin_prefix = 'tfm-rubygem-hammer_cli_'
+        $_hammer_plugin_prefix = 'tfm-rubygem-hammer_cli_'
       }
     }
     'Debian': {
-      $hammer_plugin_prefix = 'ruby-hammer-cli-'
+      $_hammer_plugin_prefix = 'ruby-hammer-cli-'
     }
     'Linux': {
       case $facts['os']['name'] {
         'Amazon': {
```

Comment:
I just checked and it looks like Amazon Linux is still qualified in the Red Hat OS family. At some point we should clean this up.

Reply:
Not me, not today, not enough :boom: ;)